libs/designer/src/lib/ui/templates/styles.less

```diff
@@ -12,7 +12,7 @@
 
 .msla-templates-wizard-tab-content {
     height: 100vh;
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
Should this be "auto" instead of scroll?  This is going to always show both horizontal and vertical scrollbars